infrastructure/modules/cloudfront/main.tf

```diff
@@ -0,0 +1,50 @@
+resource "aws_cloudfront_origin_access_control" "cloudfront_s3_oac" {
+  name                              = "CloudFront S3 OAC"
+  description                       = "Cloud Front S3 OAC"
+  origin_access_control_origin_type = "s3"
+  signing_behavior                  = "always"
+  signing_protocol                  = "sigv4"
+}
+
+resource "aws_cloudfront_distribution" "distribution" {
+  origin {
+    domain_name              = var.bucket_domain_name
+    origin_id                = var.bucket_id
+    origin_access_control_id = aws_cloudfront_origin_access_control.cloudfront_s3_oac.id
+  }
+  enabled         = true
+  is_ipv6_enabled = true
+  default_cache_behavior {
+    allowed_methods  = ["GET", "HEAD"]
+    cached_methods   = ["GET", "HEAD"]
+    target_origin_id = var.bucket_id
+    # viewer_protocol_policy = "redirect-to-https"
+
+    forwarded_values {
```

Comment:
This is [deprecated](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudfront_distribution#forwarded_values) :/.